OperatingSystem-Services/Platform-Linux/Identity Management/KeyCloak/KeyCloak.md

```diff
@@ -4,38 +4,38 @@ Keycloak is an open-source identity and access management (IAM) solution designe
 Official Website: https://www.keycloak.org/
 
 ## Getting Started (Docker)
-In order to get started we will need to install docker as this documentation focuses on using the advantages of docker to better use KeyCloak in general. If you already have docker installed you can skip to KeyCloak Install.
+In order to get started we will need to install Docker as this documentation focuses on using the advantages of Docker to better use KeyCloak in general. If you already have Docker installed you can skip to KeyCloak Install.
 
-Below is the squential command lines needed to download and install docker, more in depth documentation on the workings of docker should be found elsewhere on this repo.
+Below is the squential command lines needed to download and install Docker, more in depth documentation on the workings of Docker should be found elsewhere on this repo.
```

Comment:
Probably reword a little bit:

```suggestion
Use the following commands to install Docker, we will be leveraging the advantages of containerization in this deployment. Additional details on Docker and other containerization platforms can be found elsewhere in this repository. 
```

maybe want to mention something about how other container engines will require different steps that are not covered in something like a NOTE block
```
> [!NOTE]
> words
```